src/ontodia/diagram/model.ts

```diff
@@ -457,6 +457,7 @@ export class DiagramModel extends Backbone.Model {
             source: {id: sourceId},
             target: {id: targetId},
             vertices,
+            template: linkModel,
```

Comment:
Please add more strictly typed constructor definition to `Link` (to be able to catch invalid constructor invocations).